walker/walk_test.go

```diff
@@ -10,18 +10,22 @@ import (
 
 func Test_isIgnore(t *testing.T) {
 	for _, fp := range ignoreDirs {
-		assert.True(t, isIgnored(fp))
+		assert.True(t, isIgnored(fp, nil))
 	}
 
 	for _, fp := range ignoreSystemDirs {
-		assert.True(t, isIgnored(fp))
+		assert.True(t, isIgnored(fp, nil))
 	}
 
 	for _, fp := range library.IgnoreDirs {
-		assert.True(t, isIgnored(fp))
+		assert.True(t, isIgnored(fp, nil))
 	}
 
 	for _, fp := range []string{"foo", "foo/bar"} {
-		assert.False(t, isIgnored(fp))
+		assert.False(t, isIgnored(fp, nil))
+	}
+
+	for _, fp := range []string{"foo", "foo/bar"} {
```

Comment:
How about `/foo/bar`? This value is passed from the CLI option. I think a user usually adds the prefix `/`, but we can't predict it. We should be able to handle both.